src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.android.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Android.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		static partial void PlatformSetColor(Color color)
+		{
+			if (!BarStyle.IsSupported())
+				return;
+
+			var activity = (FormsAppCompatActivity)ToolkitPlatform.Context.GetActivity();
```

Comment:
If the activity isn't a `FormsAppCompatActivity`? Can we use the `AppCompatActivity`

Reply:
Unfortunately no. `SetStatusBarColor` method belongs directly to `FormsAppCompatActivity`